### PR TITLE
Add May newsletter

### PR DESCRIPTION
**Pull request by GregSutcliffe**
Latest newsletter, looking to merge in the next day or two, ideally.

### CODE REVIEW

File: _posts/2016-05-17-foreman-community-newsletter-may-2016.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+In another of our dives into the inner workings of Foreman, Dmitri Dolguikh
+(witlessb) gave us a look at recent changes to the smart-proxy core, using the
+changes to the puppet_proxy module needed for Puppet 4 support as an illustration.
+
+#### [Automating an R&D lab - Julien Pivotto, OSDC](https://www.youtu.be/csxeyyzZPvo)
+
+Julien gave a great talk about how to use Foreman for automating an entire lab
+at OSDC - this is definitely worth watching. Thanks Julien!
+
```
Comment (lzap) [inline _posts/2016-05-17-foreman-community-newsletter-may-2016.md:97]: Satellite 6.2 beta release might be also interesting for upstream users.

https://access.redhat.com/blogs/1169563/posts/2280311

File: _posts/2016-05-17-foreman-community-newsletter-may-2016.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+We held another community demo this month, in which we covered the usual
+community banter from me, some Remote Execution and Libvirt updates from
+Stephen and Lukas, and a lengthy chat with Dirk from NETWAYS regarding the new
+Foreman Community training course.
+
+#### [Case Study on Discovery and Custom UIs with MyHeritage](https://youtu.be/bBZzoLM7fDM)
+
+Another Case Study happened this month, and this time we chatted with Imri and
+Elad from MyHeratge about the additions they've contributed to Foreman
```
Comment (adamruzicka) [inline _posts/2016-05-17-foreman-community-newsletter-may-2016.md]: typo in `MyHeratge`, the MyHeritage folks wouldn't be happy about this

File: _posts/2016-05-17-foreman-community-newsletter-may-2016.md (hunk 1 above)
Comment (GregSutcliffe) [inline _posts/2016-05-17-foreman-community-newsletter-may-2016.md:97]: Thanks, added a short note for that.

File: _posts/2016-05-17-foreman-community-newsletter-may-2016.md (hunk 2 above)
Comment (GregSutcliffe) [inline _posts/2016-05-17-foreman-community-newsletter-may-2016.md]: Good catch, fixed.

**Comment by dLobatog** [comment]
Merged as 2ef5ba1f62e1fea30ed0270b72c21b271056afad, thanks @GregSutcliffe!

**Pull request closed without merging**